### PR TITLE
Test against OpenSearch 1.3.17.

### PR DESCRIPTION
**Pull request by dblock**
### Description

This passes tests against OpenSearch 1.x.

Since 1.x is still supported I think we should also support it in the spec. Clients can choose to ignore anything that's `x-version-removed=2.0`. This gives us a way to evolve backwards incompatible changes in 3.0.

The other option is a 1.x branch and we can have backports. It's possibly easier, and so if you think it's better, then I can PR spec and tests with `>= 2.0` separately so that the diff is smaller, and make a 1.x branch.

WDYT @nhtruong @Xtansia ?

### Issues Resolved

Part of https://github.com/opensearch-project/opensearch-api-specification/issues/84.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: spec/schemas/_core.get.yaml [hunk 1]
```diff
@@ -9,6 +9,9 @@ components:
     GetResult:
       type: object
       properties:
+        _type:
+          $ref: '_common.yaml#/components/schemas/Type'
+          x-version-removed: '2.0'
```
Comment (nhtruong) [inline spec/schemas/_core.get.yaml:14]: This is redundant since `_common.yaml#/components/schemas/Type` already has this property.

Comment (dblock) [reply spec/schemas/_core.get.yaml:14]: What's the right place for it? In `components/schemas/Type` or here?

**Comment by nhtruong** [comment]
This is fine. The changes between major versions we see now are mainly adding and removing components so this works.

File: spec/schemas/_core.get.yaml (hunk 1 above)
Comment (nhtruong) [reply spec/schemas/_core.get.yaml:14]: The entire concept of doc type was removed in 2.0 so putting it in `components/schemas/Type` makes the most sense

**Review by nhtruong — APPROVED** [review]

**Pull request merged**